app/src/main/res/layout/widget_button.xml

```diff
@@ -31,18 +32,17 @@
             android:id="@+id/widgetLabelLayout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_margin="2dp"
-            android:minHeight="32sp">
+            android:layout_margin="2dp">
 
             <TextView
                 android:id="@+id/widgetLabel"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_gravity="center"
+                android:layout_gravity="bottom|center_horizontal"
```

Comment:
As the parent LinearLayout height is `wrap_content` and no longer has a minimum height, aligning it to the bottom doesn't make any difference so this line doesn't need to be changed.